evalai/submissions.py

```diff
@@ -50,14 +50,33 @@ def push(image, phase):
     tag = image[1]
     docker_client = docker.from_env()
     try:
-        docker_client.images.get(image)
+        docker_image = docker_client.images.get(image)
+        docker_image_size = docker_image.__dict__.get('attrs').get('VirtualSize')
```

Comment:
I think this statement should go after the try-except block since this will not raise an exception if the docker image exists.